runner/config.js

```diff
@@ -17,7 +17,7 @@ var serveWaterfall = require('serve-waterfall');
 var paths = require('./paths');
 
 var HOME_DIR       = path.resolve(process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE);
-var JSON_MATCHER   = 'wct.conf.json';
+var JSON_MATCHER   = '.wct.conf.json';
```

Comment:
https://github.com/js-cli/node-findup-sync/blob/master/README.md says it uses https://github.com/jonschlinkert/micromatch so we do not need escaping neither this, nor the other dots.